src/components/ProfilePage/index.jsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState, useEffect } from "react";
+import GamesCarousel from "../GamesCarousel";
+import { useTranslation } from "react-i18next";
+import "./index.css";
+import { Col, Layout } from "antd";
+import profilePageImg from "../../images/profilePage/profilePageImg.png";
+import db from "../../firebaseConfig";
+
+const { Sider, Content } = Layout;
+const ProfilePage = () => {
+  const { t } = useTranslation();
+  const [users, setUsers] = useState([]);
+
+  const fetchUsers = async () => {
+    const usersResult = await db.collection("users").get();
+    const usersData = usersResult.docs.map((user) => user.data());
```

Comment:
you definitely don't want to be doing this; if you have 1million users for your site this is unnecessary.

instead do the search at query time.

Reply:
"filter result firebase" -> google this

Reply:
PTAL